README.md

```diff
@@ -138,7 +138,7 @@ Since the sidekick library is hosted on the same origin as the content, a static
   <body>
     <script
       type="module"
-      src="https://www.hlx.live/tools/sidekick/library/index.js"
+      src="https://www.aem.live/tools/sidekick/library/index.js"
```

Comment:
@rofe @trieloff Are we safe to point the sidekick library to `aem.live` now?

Reply:
Yes